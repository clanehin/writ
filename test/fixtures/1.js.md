# A title

Let's wrap everything in a closure:

```js
(function() {
  //:: inside
})();
```

```js
//== inside ==//

function hello(name) {
  alert("hello, " + name);
}
```
