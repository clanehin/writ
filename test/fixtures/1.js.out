(function() {
  function hello(name) {
    alert("hello, " + name);
  }
})();
